### PR TITLE
ServerStartupListener and ProfileStorage

### PR DESCRIPTION
**Pull request by isabellasalvi**
Made recommended changes to ProfileStorage and updated the Listener to include Profile info

### CODE REVIEW

File: src/main/java/codeu/model/store/basic/ProfileStore.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    }
+    return false;
+  }
+
+  /**
+   * Sets the List of Profiles stored by this ProfileStore. This should only be called once, when the data
+   * is loaded from Datastore.
+   */
+  public void setProfiles(List<Profiles> authors) {
+    this.author = author;
```
Comment (UnitingDust) [inline src/main/java/codeu/model/store/basic/ProfileStore.java]: The arguement you are passing in is a list of profiles so I would just name it "List<Profile> profiles". In the body of the method I do:
this.profiles = profiles.

File: src/main/java/codeu/controller/ServerStartupListener.java [hunk 2]
```diff
@@ -5,9 +5,11 @@
 
 import codeu.model.data.Message;
 import codeu.model.data.User;
+import codeu.model.data.Profile;
 import codeu.model.store.basic.ConversationStore;
 import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
+import codeu.model.store.basic.ProfleStore;
```
Comment (ryanluo) [inline src/main/java/codeu/controller/ServerStartupListener.java]: Typo?

File: src/main/java/codeu/controller/ServerStartupListener.java [hunk 3]
```diff
@@ -33,6 +35,9 @@ public void contextInitialized(ServletContextEvent sce) {
       List<Message> messages = PersistentStorageAgent.getInstance().loadMessages();
       MessageStore.getInstance().setMessages(messages);
 
+      List<Profile> profiles = PersistentStorageAgent.getInstance().loadProfiles();
+      ProfileStore.getInstance().setProfiles(profiles);
+
```
Comment (ryanluo) [inline src/main/java/codeu/controller/ServerStartupListener.java]: nit: extra line

File: src/main/java/codeu/model/store/basic/ProfileStore.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    }
+    return null;
+  }
+
+  /**
+   * Access the Profile object with the given UUID.
+   *
+   * @return null if the UUID does not match any existing Profile.
+   */
+  public Profile getProfile(UUID id) {
```
Comment (ryanluo) [inline src/main/java/codeu/model/store/basic/ProfileStore.java]: Isn't this the same function as above? I understand the functionality is different but you might be getting weird polymorphism?

File: src/main/java/codeu/model/store/basic/ProfileStore.java (hunk 1 above)
Comment (ryanluo) [reply src/main/java/codeu/model/store/basic/ProfileStore.java]: Yup, this line seems confusing - where is author defined?

**Pull request closed without merging**